svc/pkg/workflow/db/workflow/migrations/20240430191643_init.up.sql

```diff
@@ -0,0 +1,73 @@
+CREATE TABLE nodes (
+  id UUID PRIMARY KEY,
+  last_ping_ts INT
+);
+
+-- TODO: In the event of a node failure, clear all of the wake conditions and remove the node id. This can be
+-- done in a periodic GC service
+CREATE TABLE workflows (
+  id UUID PRIMARY KEY,
+  name TEXT NOT NULL,
```

Comment:
nit: maybe something like "workflow_name" to better imply this is the name of the implementation?